docs/content/examples/menu-filter.md

```
@@ -0,0 +1,23 @@
+
+A dropdown menu with a search field, and a submenu that filters on its own, using `DropdownMenuFilter`.
+
+</Description>
+
+<Tags />
+
+<ComponentPreview type="example"  name="MenuFilter" />
+
+<ExampleSection>
+
+### Searchable Menu
```

markdownlint-cli2 (0.23.2)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)